style.css

```diff
@@ -245,6 +223,84 @@ h3 {
   cursor: pointer;
 }
 
+/* The contact form */
+.form-content {
+  padding: 24px;
+}
```

Comment:
- [ ] As the project required, please try using Flexbox to style all the elements of this section.
 -
<img width="527" alt="image" src="https://user-images.githubusercontent.com/61919617/186193871-b1fdd93d-1cc0-4a84-aa6d-d945c84227ff.png">